debs/SPECS/wazuh-manager/debian/rules

```diff
@@ -138,6 +138,7 @@ override_dh_install:
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/debian/10
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/debian/11
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/debian/12
+	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/ol/9
```

Comment:
Could you alphabetically order this and correct the [previous one](https://github.com/wazuh/wazuh-packages/pull/2767/files#diff-84ebdeae3dd7c922738d30f63cd384cc8ac00b326d793f6325397b1b02019df1R119), please?

Reply:
You are right @lchico but we should do it in several files, I think this task should be addressed in a different issue.